src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ---
 
  * Require explicit argument when calling `Definition::setInitialPlaces()`
+ * `GuardEvent::getContext()` method has been removed. Method was not supposed to be called within guard event listeners as it always returned an empty array anyway.
+ * Add internal `HasContextTrait` trait used by all workflow event classes except by `GuardEvent` class
```

Comment:
I don't think it should be documented since everything is internal here

Reply:
I was unsure if it was worth it to add it to the changelog while it's purely internal. I've removed the line.